js_modules/dagster-ui/packages/ui-components/src/components/StyledButton.tsx

```diff
@@ -109,4 +109,6 @@ export const StyledButtonText = styled.span`
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
+  text-align: left;
+  flex: 1;
```

Comment:
I changed our `<Button>` label child to be "flex: 1" so that the right icon is always right-justified when a fixed-width or full-width is applied to the button.  Looking over a lot of our overrides in Cloud I think this will remove the need for a good chunk of them.